docs/upctl_router_list.md

```diff
@@ -0,0 +1,32 @@
+## upctl router list
+
+List routers
+
+```
+upctl router list [flags]
+```
+
+### Options
+
+```
+      --all       Show all routers.
+      --normal    Show normal routers (default). (default true)
```

Comment:
stuttering a bit here

Reply:
also; --normal is kind of a weird flag all in all and .. i'm not sure if this flag has any use? when would someone use it and how?